state/execution.go

```diff
@@ -104,15 +104,8 @@ func (blockExec *BlockExecutor) CreateProposalBlock(
 
 	evidence, evSize := blockExec.evpool.PendingEvidence(state.ConsensusParams.Evidence.MaxBytes)
 
-	// Fetch a limited amount of valid txs
```

Comment:
I deleted this line because the code doesn't actually fetch a limited amount of valid txs, it just defines a variable that is used later.